packaging/cmake/install-linux-caps-hook.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
I suggest to use `/bin/bash` instead `/bin/sh` here. Nowadays for the majority of the distribution `sh` is a symbolic link for `bash`, but there are edge cases this cannot be true.

Reply:
The script itself is 100% POSIX compliant, so `/bin/sh` is the correct choice here. `bash` is a proper superset of the POSIX Bourne shell, so running it with `bash` is still going to get us correct behavior.